runtime/src/epoch_rewards_hasher.rs

```diff
@@ -0,0 +1,17 @@
+use {siphasher::sip::SipHasher13, solana_sdk::pubkey::Pubkey, std::hash::Hasher};
+
+/// Use SipHasher13 keyed on the `seed` for calculating epoch reward partition
+pub(crate) fn create_epoch_reward_hasher(seed: u64) -> SipHasher13 {
+    SipHasher13::new_with_keys(seed, seed)
```

Comment:
this is where we'll hash in the parent blockhash

Reply:
Parent blockhash is 32 bytes. 
Here the seed is 8 bytes.
What's the convention in solana to reduce 32 byte hash to 8byte seed?

Reply:
something like this...

```suggestion
    let mut hasher = SipHasher13::new_with_keys(seed, seed);
    hasher.write(parent_blockhash.as_ref();
    hasher
```

Reply:
updated.